### PR TITLE
FAQ doc updates for removal of stored licenses in 1.11

### PR DESCRIPTION
**Pull request by ccapurso**
Support for stored licenses will be removed in Vault 1.11. This PR introduces the associated changes to the license [FAQ doc page](https://www.vaultproject.io/docs/enterprise/license/faq).

### CODE REVIEW

**Review by hghaf099 — APPROVED** [review]
LGTM

File: website/content/docs/enterprise/license/faq.mdx [hunk 1]
```diff
@@ -6,33 +6,30 @@ description: An overview of license.
...
 - [Q: How can the new ADP modules be purchased and what features are customer entitled to as part of that purchase?](#q-how-can-the-new-adp-modules-be-purchased-and-what-features-are-customer-entitled-to-as-part-of-that-purchase)
 - [Q: What is the impact to customers based on these ADP module licensing changes?](#q-what-is-the-impact-to-customers-based-on-these-adp-module-licensing-changes)
 
-### Q: When will these licensing changes be released for Vault?
+### Q: What is the product behavior change introduced by the licensing changes?
 
-The licensing and end-user license agreement (EULA) changes will be available with Vault 1.8-RC1 on June 16, 2021. Vault 1.8 is targeted to release on July 28, 2021.
+Per the [feature deprecation plans](https://www.vaultproject.io/docs/deprecation), Vault will no longer support licenses in storage. An [auto-loaded license](https://www.vaultproject.io/docs/enterprise/license/autoloading) must be used instead. If you are using stored licenses, you must migrate to auto-loaded licenses prior to upgrading to Vault 1.11
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:32]:
```suggestion
Per the [feature deprecation plans](/docs/deprecation), Vault will no longer support licenses in storage. An [auto-loaded license](/docs/enterprise/license/autoloading) must be used instead. If you are using stored licenses, you must migrate to auto-loaded licenses prior to upgrading to Vault 1.11
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 2]
```diff
@@ -44,39 +41,29 @@ No, these changes will not impact HCP Vault.
...
-When license terminates (upon grace period expiry), Vault will seal itself and customers will need a valid license in order to successfully bring-up Vault. If a valid license was not installed after license expiry, customers will need to install one, and this license will need to be auto-loaded.
+When a license expires, Vault continues to function until the license terminates. This behavior exists today and remains unchanged in Vault 1.11. The grace period, defined as the time between license expiration and license termination, is 1 day for evaluation licenses (as of 1.8), and ten years for non-evaluation licenses.
+Customers must provide a valid license before the grace period expires. This license is required to be [auto-loaded](https://www.vaultproject.io/docs/enterprise/license/autoloading). When license terminates (upon grace period expiry), Vault will seal itself and customers will need a valid license in order to successfully bring-up Vault. If a valid license was not installed after license expiry, customers will need to provide one, and this license will need to be auto-loaded.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:52]:
```suggestion
Customers must provide a valid license before the grace period expires. This license is required to be [auto-loaded](/docs/enterprise/license/autoloading). When license terminates (upon grace period expiry), Vault will seal itself and customers will need a valid license in order to successfully bring-up Vault. If a valid license was not installed after license expiry, customers will need to provide one, and this license will need to be auto-loaded.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 3]
```diff
@@ -44,39 +41,29 @@ No, these changes will not impact HCP Vault.
...
 
 ### Q: What is the impact on evaluation licenses due to this change?
 
-The 6-hour trial period for Enterprise binaries allows these particular binaries to run without a license file and will be removed as of Vault 1.8 and later. This change means that any clusters deployed with Vault 1.8 or later ENT binaries must have a valid license, and the license must be on the disk (auto-loading).
+As of Vault 1.8, any Vault cluster deployed must have a valid [auto-loaded](https://www.vaultproject.io/docs/enterprise/license/autoloading) license.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:56]:
```suggestion
As of Vault 1.8, any Vault cluster deployed must have a valid [auto-loaded](/docs/enterprise/license/autoloading) license.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 4]
```diff
@@ -44,39 +41,29 @@ No, these changes will not impact HCP Vault.
...
 
-### Q: Are there any changes to existing methods for manual license loading ( API or CLI)?
+### Q: Are there any changes to existing methods for manual license loading (API or CLI)?
 
-There are no significant changes to the API or CLI license loading with Vault 1.8.
-However, the `/sys/license` endpoint is slated for deprecation in a future release.
-Please see the question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage)
+The [`/sys/license`](https://www.vaultproject.io/api-docs/v1.10.x/system/license#install-license) and [`/sys/license/signed`](https://www.vaultproject.io/api-docs/v1.10.x/system/license#read-signed-license) endpoints have been removed as of Vault 1.11. With that said, it is no longer possible to provide a license via the `/sys/license` endpoint. License [auto-loading](https://www.vaultproject.io/docs/enterprise/license/autoloading) must be used instead.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:60]:
```suggestion
The [`/sys/license`](https://www.vaultproject.io/api-docs/v1.10.x/system/license#install-license) and [`/sys/license/signed`](https://www.vaultproject.io/api-docs/v1.10.x/system/license#read-signed-license) endpoints have been removed as of Vault 1.11. With that said, it is no longer possible to provide a license via the `/sys/license` endpoint. License [auto-loading](/docs/enterprise/license/autoloading) must be used instead.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 5]
```diff
@@ -44,39 +41,29 @@ No, these changes will not impact HCP Vault.
...
+The [`/sys/config/reload/license`](https://www.vaultproject.io/api-docs/system/config-reload#reload-license-file) endpoint can be used to reload an auto-loaded license provided as a path via an environment variable or configuration.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:61]:
```suggestion
The [`/sys/config/reload/license`](/api-docs/system/config-reload#reload-license-file) endpoint can be used to reload an auto-loaded license provided as a path via an environment variable or configuration.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 6]
```diff
@@ -86,51 +73,40 @@ The changes to licenses apply to all nodes in a cluster. The license files are n
...
-
-Also, when customers deploy new clusters to Vault 1.8 or later, a valid license must exist to upgrade successfully. This valid license must be on-disk (auto-loaded).
+Starting Vault 1.11, when customers deploy new Vault clusters, or upgrade existing Vault clusters, a valid [auto-loaded](https://www.vaultproject.io/docs/enterprise/license/autoloading) license must exist for the upgrade to be successful.
 
 ### Q: What scenarios should a customer plan for due to these license changes?
 
-- **New Cluster Deployment**: When a customer deploys new clusters to Vault 1.8 or later, a valid license must exist to successfully deploy. This valid license must be on-disk (auto-loaded).
+- **New Cluster Deployment**: When a customer deploys new clusters to Vault 1.11 or later, a valid license must exist to successfully deploy. This valid license must be on-disk ([auto-loaded](https://www.vaultproject.io/docs/enterprise/license/autoloading)).
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:80]:
```suggestion
- **New Cluster Deployment**: When a customer deploys new clusters to Vault 1.11 or later, a valid license must exist to successfully deploy. This valid license must be on-disk ([auto-loaded](/docs/enterprise/license/autoloading)).
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 7]
```diff
@@ -86,51 +73,40 @@ The changes to licenses apply to all nodes in a cluster. The license files are n
...
-
-### Q: What is the migration path for customers who want to migrate from their existing perpetually licensed binaries to the current license on storage flow?
+If a Vault cluster using a stored license is planned to be upgraded to Vault 1.11 or greater, the operator must migrate to using an auto-loaded license. The [`vault license get -signed`](https://www.vaultproject.io/docs/v1.10.x/commands/license/get) command (or underlying [`/sys/license/signed`](https://www.vaultproject.io/api-docs/v1.10.x/system/license#read-signed-license) endpoint) can be used to retrieve the license from storage in a running cluster.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:86]:
```suggestion
If a Vault cluster using a stored license is planned to be upgraded to Vault 1.11 or greater, the operator must migrate to using an auto-loaded license. The [`vault license get -signed`](/docs/v1.10.x/commands/license/get) command (or underlying [`/sys/license/signed`](/api-docs/v1.10.x/system/license#read-signed-license) endpoint) can be used to retrieve the license from storage in a running cluster.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 8]
```diff
@@ -86,51 +73,40 @@ The changes to licenses apply to all nodes in a cluster. The license files are n
...
+It is not necessary to remove the stored license entry. That will occur automatically upon startup in Vault 1.11 or greater. Prior to completing the [recommended upgrade steps](https://www.vaultproject.io/docs/upgrading), perform the following to ensure your license is properly configured:
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:87]:
```suggestion
It is not necessary to remove the stored license entry. That will occur automatically upon startup in Vault 1.11 or greater. Prior to completing the [recommended upgrade steps](/docs/upgrading), perform the following to ensure your license is properly configured:
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 9]
```diff
@@ -86,51 +73,40 @@ The changes to licenses apply to all nodes in a cluster. The license files are n
...
 
-A Vault customer currently using S3 unlocked binaries is upgrading their clusters to a pre-Vault 1.8 version, the customer must retrieve the corresponding enterprise binary and a valid license to load the license in storage (`/sys/license`). Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage)
+1. Use the command `vault license get -signed` to retrieve the license from storage of your running cluster.
+2. Put the license on disk
+3. Configure license auto-loading by specifying the [`license_path`](https://www.vaultproject.io/docs/configuration#license_path) config option or setting the [`VAULT_LICENSE`](https://www.vaultproject.io/docs/commands#vault_license) or [`VAULT_LICENSE_PATH`](https://www.vaultproject.io/docs/commands#vault_license_path) environment variable.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:91]:
```suggestion
3. Configure license auto-loading by specifying the [`license_path`](/docs/configuration#license_path) config option or setting the [`VAULT_LICENSE`](/docs/commands#vault_license) or [`VAULT_LICENSE_PATH`](/docs/commands#vault_license_path) environment variable.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 10]
```diff
@@ -86,51 +73,40 @@ The changes to licenses apply to all nodes in a cluster. The license files are n
...
+The support of licenses installed by alternative means often leads to difficulties providing the appropriate support. To provide the support expected by our customers, as we have announced in [Vault feature deprecations and plans](https://www.vaultproject.io/docs/deprecation) we are removing support for licenses in storage with Vault 1.11. This implies licensing endpoints that deal with licenses in storage will be removed, and Vault will no longer check for valid licenses in storage. This change requires that all customers have [auto-loaded](https://www.vaultproject.io/docs/enterprise/license/autoloading) licenses to upgrade to 1.11(+) successfully.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:99]:
```suggestion
The support of licenses installed by alternative means often leads to difficulties providing the appropriate support. To provide the support expected by our customers, as we have announced in [Vault feature deprecations and plans](/docs/deprecation) we are removing support for licenses in storage with Vault 1.11. This implies licensing endpoints that deal with licenses in storage will be removed, and Vault will no longer check for valid licenses in storage. This change requires that all customers have [auto-loaded](/docs/enterprise/license/autoloading) licenses to upgrade to 1.11(+) successfully.
```

File: website/content/docs/enterprise/license/faq.mdx [hunk 11]
```diff
@@ -44,39 +41,29 @@ No, these changes will not impact HCP Vault.
...
 
-When a license expires, Vault continues to function until the license terminates. This behavior exists today and remains unchanged in Vault 1.8. The grace period, defined as the time between license expiration and license termination, is 1 day for evaluation licenses (as of 1.8), and ten years for non-evaluation licenses. Customers must install a valid license before the grace period expires. This license can be autoloaded or loaded into storage using existing methods (`/sys/license`) depending on the applicable scenarios as described above.
-When license terminates (upon grace period expiry), Vault will seal itself and customers will need a valid license in order to successfully bring-up Vault. If a valid license was not installed after license expiry, customers will need to install one, and this license will need to be auto-loaded.
+When a license expires, Vault continues to function until the license terminates. This behavior exists today and remains unchanged in Vault 1.11. The grace period, defined as the time between license expiration and license termination, is 1 day for evaluation licenses (as of 1.8), and ten years for non-evaluation licenses.
```
Comment (taoism4504) [inline website/content/docs/enterprise/license/faq.mdx:51]:
```suggestion
When a license expires, Vault continues to function until the license terminates. This behavior exists today and remains unchanged in Vault 1.11. The grace period, defined as the time between license expiration and license termination, is one day for evaluation licenses (as of 1.8), and ten years for non-evaluation licenses.
```

**Review by taoism4504 — APPROVED** [review]
Just changed all links to absolute paths, but otherwise LGTM.

**Pull request merged**